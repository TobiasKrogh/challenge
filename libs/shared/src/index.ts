export * from './lib/components/message/message.component';
export * from './lib/functions/normalize-date';
export * from './lib/pipes/format-amount.pipe';
export * from './lib/pipes/format-date.pipe';
