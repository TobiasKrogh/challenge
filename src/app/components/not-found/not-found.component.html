<shared-message
  [title]="'NOT_FOUND.TITLE' | translate"
  [text]="'NOT_FOUND.TEXT' | translate"
></shared-message>
