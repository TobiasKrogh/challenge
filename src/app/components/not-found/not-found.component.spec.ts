import { ComponentFixture, TestBed } from '@angular/core/testing';
import { TranslatePipe } from '@ngx-translate/core';
import { MessageComponent } from '@challenge/shared';
import { MockPipes } from 'ng-mocks';

import { NotFoundComponent } from './not-found.component';

describe('NotFoundComponent', () => {
  let component: NotFoundComponent;
  let fixture: ComponentFixture<NotFoundComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [MessageComponent],
      declarations: [MockPipes(TranslatePipe), NotFoundComponent],
    }).compileComponents();

    fixture = TestBed.createComponent(NotFoundComponent);
    component = fixture.componentInstance;
  });

  it('should create', () => {
    fixture.detectChanges();
    expect(component).toBeTruthy();
  });

  describe('When rendering', () => {
    it('Then it matches snapshot', () => {
      fixture.detectChanges();
      expect(fixture.debugElement).toMatchSnapshot();
    });
  });
});
