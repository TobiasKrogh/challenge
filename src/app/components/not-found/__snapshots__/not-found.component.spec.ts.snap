// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundComponent When rendering Then it matches snapshot 1`] = `
DebugElement {
  "nativeNode": <div
    id="root1"
  >
    <shared-message
      class="title--with-subline"
    >
      
      
    </shared-message>
  </div>,
}
`;
